src/java.desktop/macosx/native/libawt_lwawt/awt/AWTWindow.m

```diff
@@ -1094,6 +1094,22 @@ + (AWTWindow *) lastKeyWindow {
 
 @end // AWTWindow
 
+/*
+ * Class:     sun_lwawt_macosx_CPlatformWindow
+ * Method:    nativeSetAllAllowAutomaticTabbingProperty
+ * Signature: (Z)V
+ */
+JNIEXPORT void JNICALL Java_sun_lwawt_macosx_CPlatformWindow_nativeSetAllowAutomaticTabbingProperty
+        (JNIEnv *env, jclass clazz, jboolean allowAutomaticTabbing)
+{
+    JNI_COCOA_ENTER(env);
+    if (allowAutomaticTabbing) {
+        [NSWindow setAllowsAutomaticWindowTabbing:YES];
```

Comment:
Should we access this property on the Appkit thread? Or this can be used everywhere?

Reply:
Do we plan to backport the same fix or a different one?

Reply:
We will backport the same fix with one additional  platform check.